warehouse/internal/repo/schema_test.go

```diff
@@ -73,7 +73,7 @@ func TestWHSchemasRepo(t *testing.T) {
 				require.NoError(t, err)
 
 				var (
-					now = time.Now().Truncate(time.Second).UTC()
+					now = time.Date(2023, 1, 1, 0, 0, 0, 0, time.UTC)
```

Comment:
Any reason this was updated?

Reply:
It's better to use static time for tests.